SampleBrowser/SampleBrowser/UI/Suggestion/SuggestionViewController.swift

```diff
@@ -0,0 +1,99 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import UIKit
+
+protocol SuggestionViewControllerDelegate: AnyObject {
+    func tapOnSuggestion(term: String)
+}
+
+class SuggestionViewController: UIViewController, UITableViewDelegate {
+    private var tableView: UITableView
+    private var dataSource: SuggestionDataSource!
+    private weak var delegate: SuggestionViewControllerDelegate?
+
+    private var gradientLayer: CAGradientLayer?
+    private var topIconConstraint: NSLayoutConstraint?
+
+    init() {
+        self.tableView = UITableView(frame: .zero, style: .grouped)
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    // MARK: - Life cycle
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        configureTableView()
+
+        // Showing app logo when no search is visible
+        tableView.isHidden = true
+    }
+
+    override func viewDidLayoutSubviews() {
+        super.viewDidLayoutSubviews()
+        setGradientBackground()
+        setTopConstraint()
+    }
+
+    private func setTopConstraint() {
+        let height = view.frame.height * 1 / 5
```

Comment:
Nit: this * 1 shouldn't be needed?

```suggestion
        let height = view.frame.height / 5
```